lib/bpmn/element.rb

```diff
@@ -16,7 +17,7 @@ def self.from_moddle(moddle)
         klass = "Bpmn::#{moddle["$type"].split(':').last}".constantize
```

Comment:
This should use `safe_constantize` and do a nil check instead of a general rescue.